src/plugins/ui_stateful/selection.ts

```diff
@@ -683,6 +683,12 @@ export class GridSelectionPlugin extends UIPlugin {
     ];
     handler.paste({ zones: pasteTarget }, data, { isCutOperation: true });
 
+    const selection = pasteTarget[0];
+    const col = selection.left;
+    const row = selection.top;
+    this.selection.getBackToDefault();
```

Comment:
Is it even possible to drag & drop a header if we're in the composer of a selection input ? In the codebase we sometime use `getBackToDefault` before moving the selection, and sometime we don't, it all seems very random and arbitrary ...

Reply:
- I don't think it's random - I explicitely want the selection to be updated, not the other potential listeners. I know that by the end of my manipylation, it's the selection that should have the control. Should it be done here or before calling the command ?
Or even bypass the selection processor and update the selection internally since we happen to be in the right plugin ? I'm not sure.
I agree that the 2 last solutions seem to make more sense but editing the selection directly inside the plugin seems more straightforward with the other way the plugin works. I changed it accordinly